Tropos/Sources/Controllers/CourierClient.swift

```diff
@@ -1,8 +1,21 @@
 import Courier
 
+@objc protocol TimeZone {
+    var secondsFromGMT: Int { get }
+}
+extension NSTimeZone: TimeZone {}
+
 @objc(TRCourierClient) final class CourierClient: NSObject {
     private let instance: Courier
 
+    class func channelNameForTimeZone(timeZone: TimeZone) -> String {
+        let seconds = timeZone.secondsFromGMT
+        let sign = seconds < 0 ? "minus" : "plus"
+        let hours = abs(seconds) / 3600
+        let minutes = (abs(seconds) % 3600) / 60
+        return String(format: "%@ %02d%02d", sign, hours, minutes)
```

Comment:
I wasn't sure either. Eventually this is going to be parameterized and I thought my `parameterize` implementation, which was inspired by ActiveSupport's, would do something with the dash. 

I just checked and it actually doesn't, but it also transforms spaces into dashes so the end result is the same.